src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -719,7 +719,7 @@ public void scan(HttpMessage msg, String param, String origParamValue) {
 					String expressionBodyOutput[] = {modifiedExpressionOutputUnstripped, modifiedExpressionOutputStripped};
 					boolean strippedOutput[] = {false, true};
 
-					 for (int booleanStrippedUnstrippedIndex = 0; booleanStrippedUnstrippedIndex < 2 && !sqlInjectionFoundForUrl; booleanStrippedUnstrippedIndex++) {
+					for (int booleanStrippedUnstrippedIndex = 0; booleanStrippedUnstrippedIndex < 2 && !sqlInjectionFoundForUrl; booleanStrippedUnstrippedIndex++) {
```

Comment:
Formatting change should be submitted as a separate PR: https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting